types/artifacts.d.ts

```diff
@@ -415,20 +415,24 @@ declare global {
         /** The raw height attribute of the image element. CSS images will be set to the empty string. */
         attributeHeight: string;
         /** The CSS width property of the image element. */
-        cssWidth?: string | undefined;
+        cssWidth: string | undefined;
```

Comment:
paging dr. type @brendankenny 

should we try to make this `cssWidth?: string` ?

Reply:
I would typically prefer `cssWidth: string | undefined` when the property is important as it ensures explicit definition that you're not forgetting something when creating the object and nearly all consumption of the property will be identical since we typically avoid `in`.

I would not expect there to be a benefit here. I am curious if @brendankenny disagrees :)

Reply:
I agree with @patrickhulce for the better developer experience/improving maintainability, but I think technically it's `cssWidth?: string`. When artifacts are `JSON.stringify()`ed, any properties that are set to `undefined` are dropped, so audits need to treat it as if it were optional.

Another option is `string | null` (which will survive the JSON roundtrip) but that might be a bridge too far, I don't know :)

Reply:
thanks for the clarifications! I changed it to `cssWidth/Height?: string`